README.md

```diff
@@ -1,25 +1,35 @@
 - `psibase`, the command-line utility for pushing transactions and booting a chain
-- psibase services
+- psibase system applications 
+- psibase example user applications
 - `psidk`, the service development kit
 
-## About Page, Documentation, and Demo
+## Demo deployment
 
-The [Demonstration Chain](https://psibase.io/) hosts both the [About Page](https://about.psibase.io/) and the [Documentation](https://doc-sys.psibase.io/).
+A demonstration deployment is currently hosted at [psibase.io](https://psibase.io) to showcase what psibase infrastructure could look like for end users. The applications available on this demo are sample applications and are not intended to be used for a production deployment.
 
-## Binary Release
+## Technical documentation
+
+Documentation is hosted on the [doc-sys applet](https://doc-sys.psibase.io/) of our demo deployment.
+
+## Running psibase
+
+If you want to operate a node, build services, or explore psibase's command-line utility, we recommend one of the following methods:
+1. Installing a binary release, the latest of which can be found in our [Rolling Release](https://github.com/gofractally/psibase/releases). We provide binary packages for both Ubuntu 20.04 and Ubuntu 22.04.
+2. Installing our psinode [Docker Image](https://github.com/gofractally/psinode-docker-image/pkgs/container/psinode). Our docker image can be used to run psibase on any operating system that supports Docker. Currently we do not build an image for the arm architecture, and therefore the latest [Apple Silicon](https://support.apple.com/en-us/HT211814) based devices are not supported. 
 
-If you want to operate a node, build services, or explore psibase's command-line utility, we recommend installing a [Binary Release](https://github.com/gofractally/psibase/releases). We provide a single binary package which should run on most recent Linux distributions. See the [installation instructions](https://doc-sys.psibase.io/linux.html).
 
 ## Building this repo from source
 
-See the architecture-specific instructions below.
+We only support developing natively on Ubuntu 20.04 and Ubuntu 22.04. Some of our developers have very strong success with Ubuntu on WSL2, and also with Docker.
+
+To build this repo from source, your environment first needs to be set up. To set up your environment, follow the architecture-specific instructions found in either the [Ubuntu 20.04 dockerfile](https://github.com/gofractally/image-builders/blob/main/docker/ubuntu-2004-builder.Dockerfile) or the [Ubuntu 22.04 dockerfile](https://github.com/gofractally/image-builders/blob/main/docker/ubuntu-2204-builder.Dockerfile).  Alternatively, if you would prefer to use a containerized development environment (for example to isolate the dependencies, or to develop on operating systems other than Ubuntu 20.04 and Ubuntu 22.04 such as Windows or Mac), see the instructions found in the [Psibase Contributor repository](https://github.com/gofractally/psibase-contributor).
```

Comment:
> follow the architecture-specific instructions found in either the [Ubuntu 20.04 dockerfile]

The docker files aren't exactly instructions to follow.  They're more like an example of one specific way to set up the environment.  There are a number of commands that are not suitable for setting up a non-containerized environment, so blindly copying it isn't a good idea.

> see the instructions found in the [Psibase Contributor repository]

I would cut out "the instructions found in." It doesn't really add anything.